Portal/src/Datahub.Portal/Templates/Onboarding/OnboardingClient.razor

```diff
@@ -2,15 +2,15 @@
 
 <h3>Onboarding Application</h3>
 <h4>Onboarding Request</h4>
-<p>@ApplicationParameters.App.Client_Contact_Name </p>
+<p>@ApplicationParameters.Client_Contact_Name </p>
 <p>Thank you for completing the DataHub Onboarding Form. We’ve received your application and a DataHub team member will be in touch with you shortly.</p>
 <p>Please contact datahub@nrcan-rncan.gc.ca if you have questions or require additional information.</p>
```

Comment:
This email address should be updated to the SSC version here and in the French section.